backend/src/nysenate-api.js

```diff
@@ -13,7 +13,7 @@ export const legApi = (path, params = {}) => {
 
 export const billsFromYear = async(year) => {
   // First request with no offset
-  let firstResponse = await fetch(legApi(`bills/${year}`));
+  let firstResponse = await fetch(legApi(`bills/${year}`, { full: true }));
```

Comment:
This is a lot of added data to the main page, which is already getting too much.

I think this should go on the page for an individual bill - `Bill.jsx`. You can add the field & it's not a big deal since it's only one item requested.

Reply:
good call, will move this to individual bill page